api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -34,10 +34,7 @@ message VmConfig {
   // The Wasm code that Envoy will execute.
   config.core.v3.AsyncDataSource code = 3;
 
-  // The Wasm configuration used in initialization of a new VM
-  // (proxy_on_start). `google.protobuf.Struct` is serialized as JSON before
-  // passing it to the plugin. `google.protobuf.BytesValue` and
-  // `google.protobuf.StringValue` are passed directly without the wrapper.
+  // The Wasm configuration used in initialization of a new VM (proxy_on_start).
```

Comment:
The original description should be kept?

Reply:
Done.